### ISSUE TITLE
onstar2mqtt failing at startup

### ISSUE THREAD

**Issue by ehran** [michaelwoods/onstar2mqtt#208]
I was wondering why this was not working anymore for some time and checked my syno docker and the service keeps failing to start. logs show an error 400 after the command "Requesting vehicles" starting with 
error: Main function error. {"error":{"message":"Request Failed with status 400 - Bad Request","request":{" etc...

**Comment by BennyDaBee** [comment]
@ehran Do you use docker or do you start with NPM?

**Comment by nodiaque** [comment]
I just had this problem. Be sure you enter all information, including the pin code. I had this problem when there was something missing like pin code

**Comment by michaelwoods** [comment]
Try to update to the latest image (the API is sensitive to the library version being current) or at least add some logs to help resolve the issue.

**Comment by ehran** [comment]
Hi all,
So to resolve the issue last time I had to delete the container, download the latest version and create the container again. I m using Docker on Synology with a user interface. I have tried all I could to get the container to update but it doesnt want to ... any idea why this is happening ? I mention that I havent changed anything in the parameters or credentials. Was working till yesterday for sure. Unsure for this morning.
thanks !
